conf/app.routes

```diff
@@ -91,6 +91,16 @@ GET        /nominated-partner-check-answers   uk.gov.hmrc.plasticpackagingtax.re
 GET        /partner-check-answers/:partnerId  uk.gov.hmrc.plasticpackagingtax.registration.controllers.partner.PartnerCheckAnswersController.partner(partnerId: String)
 POST       /partner-check-answers             uk.gov.hmrc.plasticpackagingtax.registration.controllers.partner.PartnerCheckAnswersController.submit()
 
+GET        /partnerships/partners-list  uk.gov.hmrc.plasticpackagingtax.registration.controllers.partnership.PartnershipPartnersListController.displayPage()
+GET        /partnerships/other-partners/contact-name uk.gov.hmrc.plasticpackagingtax.registration.controllers.partnership.PartnershipOtherPartnerContactNameController.displayPage()
+POST       /partnerships/other-partners/contact-name uk.gov.hmrc.plasticpackagingtax.registration.controllers.partnership.PartnershipOtherPartnerContactNameController.submit()
+GET        /partnerships/other-partners/email-address  uk.gov.hmrc.plasticpackagingtax.registration.controllers.partnership.PartnershipOtherPartnerEmailAddressController.displayPage()
+POST       /partnerships/other-partners/email-address  uk.gov.hmrc.plasticpackagingtax.registration.controllers.partnership.PartnershipOtherPartnerEmailAddressController.submit()
+GET        /partnerships/other-partners/phone-number  uk.gov.hmrc.plasticpackagingtax.registration.controllers.partnership.PartnershipOtherPartnerPhoneNumberController.displayPage()
+POST       /partnerships/other-partners/phone-number  uk.gov.hmrc.plasticpackagingtax.registration.controllers.partnership.PartnershipOtherPartnerPhoneNumberController.submit()
+GET        /partnerships/check-answers-other-partner uk.gov.hmrc.plasticpackagingtax.registration.controllers.partnership.PartnershipOtherPartnerCheckAnswersController.displayPage()
+POST       /partnerships/check-answers-other-partner uk.gov.hmrc.plasticpackagingtax.registration.controllers.partnership.PartnershipOtherPartnerCheckAnswersController.submit()
+
```

Comment:
We need to thrash out the URLs for a partnership registration so that they are consistent.